.distignore

```diff
@@ -21,3 +26,10 @@ tests
 *.xml
 *.yml
 *.zip
+*.scss
+
+# Newfold src files
+vendor/newfold-labs/wp-module-ecommerce/src
+vendor/newfold-labs/wp-module-help-center/src
+vendor/newfold-labs/wp-module-onboarding/src
+vendor/newfold-labs/wp-module-patterns/src
```

Comment:
We should probably implement a pattern like this:
```suggestion
vendor/newfold-labs/wp-module-*/src
```

Reply:
I thought about this, but there are still some modules that need the src directory. One example is the data module, though it would be simple enough to rename the `src` directory in that module to `includes` since that follows the pattern of all other modules better. I also see a src dir in the runtime module. Once we fix both of those I think we could update this with the pattern though.